src/Compilers/Core/MSBuildTask/Microsoft.Build.Tasks.CodeAnalysis.csproj

```diff
@@ -66,9 +66,8 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Build.Framework" Version="$(RefOnlyMicrosoftBuildFrameworkVersion)" ExcludeAssets="Runtime" />
     <PackageReference Include="Microsoft.Build.Tasks.Core" Version="$(RefOnlyMicrosoftBuildTasksCoreVersion)" ExcludeAssets="Runtime" />
-    <PackageReference Include="System.IO.Pipes.AccessControl" Version="$(SystemIOPipesAccessControlVersion)" Condition="'$(TargetFramework)' == 'netcoreapp3.1'" />
```

Comment:
`System.IO.Pipes.AccessControl` became inbox in .NET 6 (you will see that [there is no stable 6.0.0 version](https://www.nuget.org/packages/System.IO.Pipes.AccessControl/#versions-body-tab)).

Reply:
Don't we still need this for `net472` though?

Reply:
The .NET Framework assemblies of the package consist mostly of type forwarders.

![image](https://user-images.githubusercontent.com/12659251/205760581-d9cbb482-3343-416c-97e4-5e4b44b8cff1.png)